addons/binding/org.openhab.binding.netatmo/README.md

```diff
@@ -180,25 +180,25 @@ Number Netatmo_Indoor2_Temperature "Temperature" { channel = "netatmo:NAModule4:
 
 **Supported channels for the additional indoor module:**
 
-| Channel ID          | Item Type | Description                                              |
-|---------------------|-----------|----------------------------------------------------------|
-| Co2                 | Number    | Air Quality in ppm                                       |
-| Temperature         | Number    | Current temperature                                      |
-| TempTrend           | String    | Temperature Evolution Trend (up, down, stable)           |
-| Humidity            | Number    | Current humidity in %                                    |
-| Humidex             | Number    | Computed Humidex index                                   |
-| HeatIndex           | Number    | Computed Heat Index                                      |
-| Dewpoint            | Number    | Computed Dewpoint Temperature                            |
-| DewpointDepression  | Number    | Computed Dewpoint Depression                             |
-| MinTemp             | Number    | Minimum Temperature on current day                       |
-| MaxTemp             | Number    | Maximum Temperature on current day                       |
-| DateMinTemp         | DateTime  | Date when minimum temperature was reached on current day |
-| DateMaxTemp         | DateTime  | Date when maximum temperature was reached on current day |
-| TimeStamp           | DateTime  | Timestamp when data was measured                         |
-| LastMessage         | DateTime  | Last Message emitted by the module                       |
-| LowBattery          | Switch    | Low Battery                                              |
-| BatteryVP           | Number    | Battery Level                                            |
-| RfStatus            | Number    | Signal Strength (0 for no signal, 1 for weak, 2 for average, 3 for good or 4 for excellent) |
+| Channel ID          | Item Type            | Description                                              |
+|---------------------|----------------------|----------------------------------------------------------|
+| Co2                 | Number:Dimensionless | Air Quality in ppm                                       |
+| Temperature         | Number:Temperature   | Current temperature                                      |
+| TempTrend           | String               | Temperature Evolution Trend (up, down, stable)           |
+| Humidity            | Number:Dimensionless | Current humidity in %                                    |
```

Comment:
Please remove `in %`.

There are more units left in the tables. Please remove the units throughout the complete file.